### DISCUSSION TITLE
Username already taken and Character does not exist

### DISCUSSION THREAD

**Discussion by RantarR** [evennia/evennia#2548] [Evennia Questions] [answered]
Hello!
I deleted my test character.
But now i cant recreate him ("Username already taken").
I cant log him on either ("Character does not exist").

How can i recreate him? He got too many redundant attributes so
i wanted to make a "fresh" version.

**Comment by Griatch** [comment, marked as answer]
You deleted the `Character`, not the `Account` - the two are separate in the sense that the `Account` puppets the `Character`. In default setup, the system will create both at the same time with the same name (see "Accounts" and "Objects" respectively in the documentation). 

You can use the `charcreate` command to (re)create a new character. You can also delete an Account by using `delaccount` (don't delete your superuser though). 

If you haven't a lot of stuff in your database yet (which you shouldn't have if you are just trying stuff out), you can of course also easily just stop evennia, delete the `mygame/server/evennia.db3` database file and then run `evennia migrate` and `evennia start` again to recreate the database and create a new superuser with a fresh db.

**Reply by RantarR** [reply]
Thank you for the detailed answer. This has helped me a lot.
